File: atividade_C/atividade04.c
#include <stdio.h>

int idade,maior_idade;
int main()
{

    for(int i= 1; i<=10; i++)
    {
        printf("Informe a idade: \n");
        scanf("%d", &idade);

        if(idade >= 18)
        {
          maior_idade++;
        }
    }
    
    printf("Temos %d usuário(s) maiores de 18 anos\n",maior_idade);

return 0;
}

File: 2024/atividade_py/atividade04.py
maior_idade = 0;

for i in range(1,11):
    
    print(f"Informe a idade do {i}º usuário: ",end="");
    idade = int(input());

    if idade >= 18:
        
        maior_idade+= 1;

print(f"{maior_idade} Usuários são maiores de 18 anos");